engine/gamesys/src/gamesys/resources/res_texture.cpp

```diff
@@ -138,8 +138,7 @@ namespace dmGameSystem
             {
                 num_mips = MAX_MIPMAP_COUNT;
                 output_format = dmGraphics::GetSupportedCompressionFormat(context, output_format, image->m_Width, image->m_Height);
-                bool result = dmGraphics::Transcode(path, image, image_desc->m_DDFImage->m_Count, output_format, image_desc->m_DecompressedData, image_desc->m_DecompressedDataSize, &num_mips);
-                if (!result)
+                if (!dmGraphics::Transcode(path, image, image_desc->m_DDFImage->m_Count, output_format, image_desc->m_DecompressedData, image_desc->m_DecompressedDataSize, &num_mips))
```

Comment:
I think it's a bit strange to pass in "m_Count" (which was previously always 1, since it was a single blob), and also num_mips. I think the m_Count should suffice.